Dockerfile

```diff
@@ -1,19 +1,30 @@
-# syntax=docker/dockerfile:1
 FROM mcr.microsoft.com/devcontainers/javascript-node:0-18 AS build
-ARG REACT_APP_BACKEND_URL
```

Comment:
don't remove ARG. ARG and ENV can work together

https://vsupalov.com/docker-arg-env-variable-guide/#arg-and-env


> What if you want to set ENV variables dynamically during build? When building an image, the only thing you can provide are ARG values. You can’t provide values for ENV variables directly. However, both ARG and ENV can work together. You can use ARG to set the default values of ENV vars in your Dockerfile.

Reply:
@ThomasCardin you had resolved this conversation without comment but I still see ARG as removed.

Reply:
@ThomasCardin please address this conversation

Reply:
I saw you have both ARG and ENV now